chicagoland.pandemicresponsecommons.org/manifests/hatchery/hatchery.json

```diff
@@ -5,20 +5,35 @@
   "sidecar": {
     "cpu-limit": "1.0",
     "memory-limit": "256Mi",
-    "image": "quay.io/cdis/gen3fuse-sidecar:0.2.3",
-    "env": {"NAMESPACE":"default", "HOSTNAME": "chicagoland.pandemicresponsecommons.org"},
+    "image": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/gen3fuse-sidecar:2021.03",
+    "env": {
+      "NAMESPACE": "default",
+      "HOSTNAME": "chicagoland.pandemicresponsecommons.org"
+    },
     "args": [],
-    "command": ["/bin/bash", "/sidecarDockerrun.sh"],
-    "lifecycle-pre-stop": ["su", "-c", "echo test", "-s", "/bin/sh", "root"]
+    "command": [
+      "/bin/bash",
+      "/sidecarDockerrun.sh"
+    ],
+    "lifecycle-pre-stop": [
+      "su",
+      "-c",
+      "echo test",
+      "-s",
+      "/bin/sh",
+      "root"
+    ]
   },
   "containers": [
     {
       "target-port": 8787,
       "cpu-limit": "1.0",
       "memory-limit": "1024Mi",
-      "name": "R Studio-3.5",
+      "name": "R Studio 3.5",
       "image": "quay.io/cdis/rstudio:master",
-      "env": {"DISABLE_AUTH": "true"},
+      "env": {
+        "DISABLE_AUTH": "true"
```

Comment:
anyone knows what this does?

Reply:
i think this is bypassing the login page of the rstudio container